src/apb/engine.py

```diff
@@ -208,9 +248,25 @@ def cmdrun_init(**kwargs):
     bindable = kwargs['bindable']
     async = kwargs['async']
     params = kwargs['params']
+    skip = kwargs['skip']
 
     description = "This is a sample application generated by apb init"
 
+    # Ensure that command line options are acceptable
+    for skip_option in skip:
+        if skip_option not in SKIP_OPTIONS:
+            raise Exception(
+                "Error: [%s] is not a valid option to skip. The following are skippable options: %s" %
+                (skip_option, SKIP_OPTIONS)
+            )
```

Comment:
The idea I'm shooting for is instead of:
```
--skip provision
```
They can type in
```
--skip-provision
```
While it seems silly for a single dash but it makes it less error prone from the users perspective.